README.md

```diff
@@ -1,6 +1,42 @@
-# get-country
-[![Travis CI](https://travis-ci.org/busrapidohq/get-country.svg?branch=master)](https://travis-ci.org/busrapidohq/get-country) [![Codecov](https://img.shields.io/codecov/c/github/busrapidohq/get-country/master.svg)](https://codecov.io/gh/busrapidohq/get-country) [![npm](https://img.shields.io/npm/dm/get-country.svg)](https://www.npmjs.com/package/get-country) [![npm version](https://img.shields.io/npm/v/get-country.svg)](https://www.npmjs.com/package/get-country) [![npm dependencies](https://david-dm.org/busrapidohq/get-country.svg)](https://david-dm.org/busrapidohq/get-country) [![npm dev dependencies](https://david-dm.org/busrapidohq/get-country/dev-status.svg)](https://david-dm.org/busrapidohq/get-country#info=devDependencies)  
+# country-code
+[![Travis CI](https://travis-ci.org/busrapidohq/country-code.svg?branch=master)](https://travis-ci.org/busrapidohq/country-code) [![Codecov](https://img.shields.io/codecov/c/github/busrapidohq/country-code/master.svg)](https://codecov.io/gh/busrapidohq/country-code) [![npm](https://img.shields.io/npm/dm/country-code.svg)](https://www.npmjs.com/package/country-code) [![npm version](https://img.shields.io/npm/v/country-code.svg)](https://www.npmjs.com/package/country-code) [![npm dependencies](https://david-dm.org/busrapidohq/country-code.svg)](https://david-dm.org/busrapidohq/country-code) [![npm dev dependencies](https://david-dm.org/busrapidohq/country-code/dev-status.svg)](https://david-dm.org/busrapidohq/country-code#info=devDependencies)
 🗺 Get ISO 3166-1 alpha-3 country code for geographic coordinates.
 
-`test/fixtures/cities.json` was exctracted from data licensed under a Creative Commons Attribution 3.0 License,
-see [cc3](http://creativecommons.org/licenses/by/3.0/) from geonames.org
+## Install
+
+```
+$ npm install --save country-code
+$ npm install --save world-countries-boundaries-5m
+```
+We recommand you to install one of the `world-countries-boundaries-*` packages
+
+## Usage
+
+```javascript
+const countryCode = require('country-code');
+
+// Load a GeoJSON with country boundaries
+countryCode.use(require('world-countries-boundaries-1m')());
+
+// query a point
+countryCode.get(42.50779, 1.52109);
+// > 'AD'
+```
+
+## API
+### .use(geoJson)
+Load a geoJson and preprocess the data to speed up future queries
+- *geoJson*, Object, a GeoJson FeatureCollection. You can get one with customizable accuracy at https://github.com/busrapidohq/world-country-boundaries
```

Comment:
### use(geoJson)

Pre computes an R-Tree from a GeoJSON Object and uses the data future queries.

#### geoJson

Type: `object`

A valid GeoJSON FeatureCollection, each feature must have .properties.ISO_A3